user-guide/Cloud_Platform/CloudAdminApp/Accessing_the_Admin_app.md

```diff
@@ -4,9 +4,29 @@ uid: Accessing_the_Admin_app
 
 # Accessing the Admin app
 
-1. Go to <https://admin.dataminer.services/>.
+## Direct URL
+
+The simplest way to access the app is by browsing directly to the app URL:
+
+1. Go to <https://admin.dataminer.services>.
 
 1. Log in as detailed under [Logging on](xref:Logging_on_to_the_DataMiner_Cloud_Platform#logging-on).
 
-> [!TIP]
-> From DataMiner 10.2.9/10.3.0 onwards, you can navigate directly to the Admin app from DataMiner Cube by selecting *Cloud Admin* in the *Apps* pane.
+## Via dataminer.services
+
+You can also go to the app via dataminer.services:
+
+1. Go to [dataminer.services](https://dataminer.services)
+
+1. Find the DMS in the list and click the *Admin* icon.
```

Comment:
What about the Admin icon higher up under Apps? 
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/94605575/44db59a7-a3ee-477f-b0bc-5b6b2f16776c)

Perhaps we can give the user the choice here? 

```suggestion
1. If you want to open the Admin app to manage a specific DMS, find the DMS in the list and click the *Admin* icon.

   If you are not interested in a specific DMS, click the Admin icon under *Apps*.
```

Reply:
Yes, sounds good :) One opens the Organization overview, the other the DMS overview of the one selected indeed.